lib/phoenix/socket/serializers/v2_json_serializer.ex

```diff
@@ -10,8 +10,8 @@ defmodule Phoenix.Socket.V2.JSONSerializer do
 
   @impl true
   def fastlane!(%Broadcast{payload: {:binary, data}} = msg) do
-    topic_size = byte_size(msg.topic)
-    event_size = byte_size(msg.event)
+    topic_size = byte_size!(msg.topic, :topic, 256)
```

Comment:
Shouldn't the max here be 255?

Reply:
`byte_size!` does `< max`, so we are enforcing 255, but I think 255 with `<=` would be clearer?